paasta_tools/marathon_tools.py

```diff
@@ -1524,3 +1524,19 @@ def take_up_slack(client: MarathonClient, app: MarathonApp) -> None:
         log.info("Scaling %s down from %d to %d instances to remove slack." %
                  (app.id, app.instances, app.instances - slack))
         client.scale_app(app_id=app.id, instances=(app.instances - slack), force=True)
+
+
+def marathon_dashboard_base_url(job_config, system_paasta_config):
+    cluster = system_paasta_config.get_cluster()
+    try:
+        links = system_paasta_config.get_dashboard_links().get(cluster).get('Marathon RO')
+    except KeyError:
+        links = ''
+    if isinstance(links, list):
+        pinned_shard = job_config.get_marathon_shard()
+        if pinned_shard:
+            return links[pinned_shard]
+        else:
+            service_instance = compose_job_id(job_config.service, job_config.instance)
```

Comment:
I'm really concerned we have to do this in a second place. 
I think ideally we would only ever have the key= setting for rendezvous_hash in a single place ever.

I expected this code to look like this:

shard = lookup_shard(job_config)
return links[shard]

def lookup_shard(job_config):
... I guess this needs some sort of "choises".

...
```

I guess what I don't like is the possibility that this shard lookup can be "out of sync" with that the actual shard is.

This is not a blocker.